Jenkinsfile

```diff
@@ -104,7 +114,8 @@ if( buildParams.BUILD_OS != "all" ){
 
 // Using .each{} hits "a CPS-transformed closure is not yet supported (JENKINS-26481)"
 for( label in build_nodes) {
-  stage_unit_test(label)
-  stage_rpmbuild(label)
-  stage_integration(label)
+//stage_unit_test(label)
+//stage_rpmbuild(label)
+//stage_integration(label)
+  stage_clean_rpms(label)
 }
```

Comment:
I commented out the stage_xxx steps because they are unrelated to what this branch does. I think it is a waste of resources to have Jenkins run builds and tests on code not relevant to this branch.